pydoctor/epydoc2stan.py

```diff
@@ -844,6 +844,18 @@ def get_parsed_type(obj: model.Documentable) -> Optional[ParsedDocstring]:
 
     return None
 
+def format_toc(obj: model.Documentable) -> Optional[Tag]:
+    # Load the parsed_docstring if it's not already done. 
+    if not obj.parsed_docstring and obj.docstring:
+        obj.parsed_docstring = parse_docstring(obj, obj.docstring, obj)
```

Comment:
This is not tested because the of order of execution of methods (but it's there just in case basically because we never know), utimately we'de like all docstrings to be parsed at the same time, in a specific step of the process.